configuration/configuration.go

```diff
@@ -1103,14 +1098,12 @@ func (c *ConfigurationData) GetUserDeactivationFetchLimit() int {
 
 // GetUserDeactivationInactivityNotificationPeriodDays returns the number of days of inactivity before notifying the user of the imminent account deactivation
 func (c *ConfigurationData) GetUserDeactivationInactivityNotificationPeriodDays() time.Duration {
-	//return time.Duration(c.v.GetFloat64(varUserDeactivationInactivityNotificationPeriodDays)) * 24 * time.Hour
-	return time.Minute * 30
+	return time.Duration(c.v.GetFloat64(varUserDeactivationInactivityNotificationPeriodDays)) * 24 * time.Hour
```

Comment:
This `GetFloat64` was used for manual_test, before it was `GetInt`.  So we should replace with `GetInt`.